homeassistant/components/renson_endura_delta/const.py

```diff
@@ -0,0 +1,218 @@
+    name="Ventilation level raw",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+CURRENT_LEVEL_DESC = SensorEntityDescription(
+    key="CURRENT_LEVEL_FIELD",
+    name="Ventilation level",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+CURRENT_AIRFLOW_EXTRACT_DESC = SensorEntityDescription(
+    key="CURRENT_AIRFLOW_EXTRACT_FIELD",
+    name="Total airflow out",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="m³/h",
+)
+
+CURRENT_AIRFLOW_INGOING_DESC = SensorEntityDescription(
+    key="CURRENT_AIRFLOW_INGOING_FIELD",
+    name="Total airflow int",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="m³/h",
+)
+
+OUTDOOR_TEMP_DESC = SensorEntityDescription(
+    key="OUTDOOR_TEMP_FIELD",
+    name="Outdoor air temperature",
+    device_class="temperature",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement=TEMP_CELSIUS,
+)
+
+INDOOR_TEMP_DESC = SensorEntityDescription(
+    key="INDOOR_TEMP_FIELD",
+    name="Extract air temperature",
+    device_class="temperature",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement=TEMP_CELSIUS,
+)
+
+FILTER_REMAIN_DESC = SensorEntityDescription(
+    key="FILTER_REMAIN_FIELD",
+    name="Filter change",
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="days",
+)
+
+HUMIDITY_DESC = SensorEntityDescription(
+    key="HUMIDITY_FIELD",
+    name="Relative humidity",
+    device_class=DEVICE_CLASS_HUMIDITY,
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement="%",
+)
+
+MANUAL_LEVEL_DESC = SensorEntityDescription(
+    key="MANUAL_LEVEL_FIELD",
+    name="Manual level",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+TIME_AND_DATE_DESC = SensorEntityDescription(
+    key="MANUAL_LEVEL_FIELD",
+    name="System time",
+    device_class=DEVICE_CLASS_TIMESTAMP,
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+BREEZE_TEMPERATURE_DESC = SensorEntityDescription(
+    key="BREEZE_TEMPERATURE_FIELD",
+    name="Breeze temperature",
+    device_class=TEMP_CELSIUS,
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement=TEMP_CELSIUS,
+)
+
+BREEZE_LEVEL_DESC = SensorEntityDescription(
+    key="BREEZE_LEVEL_FIELD",
+    name="Breeze level",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+DAYTIME_DESC = SensorEntityDescription(
+    key="DAYTIME_FIELD",
+    name="Start day time",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+
+NIGHTTIME_DESC = SensorEntityDescription(
+    key="NIGHTTIME_FIELD",
+    name="Start night time",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+DAY_POLLUTION_DESC = SensorEntityDescription(
+    key="DAY_POLLUTION_FIELD",
+    name="Day pollution level",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+NIGHT_POLLUTION_DESC = SensorEntityDescription(
+    key="NIGHT_POLLUTION_FIELD",
+    name="Night pollution level",
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+CO2_THRESHOLD_DESC = SensorEntityDescription(
```

Comment:
These shouldn't be all standalone constants. Move them to the `sensor.py` file and make them a tuple that contains them all. Just iterate over them inside your `async_setup_entry` and create entities.